File: Prism.Scripting/src/Prism/Core/Time.cs
using System;

namespace Prism
{
    public static class Time
    {
        public static float DeltaTime { get { unsafe { return InternalCalls.Prism_Time_GetDeltaTime(); } } }
        public static float UnscaledDeltaTime { get { unsafe { return InternalCalls.Prism_Time_GetUnscaledDeltaTime(); } } }
        public static float GameTime { get { unsafe { return InternalCalls.Prism_Time_GetTime(); } } }
        public static float UnscaledTime { get { unsafe { return InternalCalls.Prism_Time_GetUnscaledTime(); } } }
        public static float FixedDeltaTime { get { unsafe { return InternalCalls.Prism_Time_GetFixedDeltaTime(); } } set { unsafe { InternalCalls.Prism_Time_SetFixedDeltaTime(value); } } }
        public static UInt64 FrameCount { get { unsafe { return InternalCalls.Prism_Time_GetFrameCount(); } } }
        public static float TimeScale { get { unsafe { return InternalCalls.Prism_Time_GetTimeScale(); } } set { unsafe { InternalCalls.Prism_Time_SetTimeScale(value); } } }
    }
}

File: PrismEditor/Assets/scripts/Python/Prism/Core/Time.py
import PrismNative as _Prism


class TimeMeta(type):
    @property
    def DeltaTime(cls) -> float:
        return _Prism.Prism_Time_GetDeltaTime()

    @property
    def UnscaledDeltaTime(cls) -> float:
        return _Prism.Prism_Time_GetUnscaledDeltaTime()

    @property
    def Time(cls) -> float:
        return _Prism.Prism_Time_GetTime()

    @property
    def UnscaledTime(cls) -> float:
        return _Prism.Prism_Time_GetUnscaledTime()

    @property
    def FixedDeltaTime(cls) -> float:
        return _Prism.Prism_Time_GetFixedDeltaTime()

    @property
    def FrameCount(cls) -> int:
        return _Prism.Prism_Time_GetFrameCount()

    @property
    def TimeScale(cls) -> float:
        return _Prism.Prism_Time_GetTimeScale()

    @TimeScale.setter
    def TimeScale(cls, value: float) -> None:
        _Prism.Prism_Time_SetTimeScale(value)

    @FixedDeltaTime.setter
    def FixedDeltaTime(cls, value: float) -> None:
        _Prism.Prism_Time_SetFixedDeltaTime(value)


class Time(metaclass=TimeMeta):
    pass
